inspectit-oce-core/src/main/java/rocks/inspectit/oce/core/rocks/inspectit/oce/core/config/DirectoryPropertySource.java

```diff
@@ -0,0 +1,146 @@
+package rocks.inspectit.oce.core.rocks.inspectit.oce.core.config;
```

Comment:
should we move everything related to the property sources to the  `config.propertysource` sub-package just to organize a bit..